### PR TITLE
Fix #14: uninitialized Confidential App Generator

### LINKED ISSUE #14: uninitialized constant DoorkeeperSequel::ConfidentialApplicationsGenerator
Either I am missing something or there is an error in this gem. Because, I cannot generate `pkce` and `confidental_applications` migrations. Tested on Windows and linux.

Result:

```
$ rake doorkeeper_sequel:generate:confidential_applications --trace
/d/Ruby25-x64/bin/rake: line 4: {#: command not found
I, [2018-10-28T12:18:48.278405 #16532]  INFO -- sentry: ** [Raven] Raven 2.7.4 ready to catch errors
** Invoke doorkeeper_sequel:generate:confidential_applications (first_time)
** Execute doorkeeper_sequel:generate:confidential_applications
I, [2018-10-28T12:18:48.689970 #16532]  INFO -- sentry: ** [Raven] Sending event 26e04e795503450dbe144ceeed00e19f to Sentry
E, [2018-10-28T12:18:49.478449 #16532] ERROR -- sentry: ** [Raven] Unable to record event with remote Sentry server (Raven::Error - the server responded with status 429 Error in headers is: Creation of this event was denied due to rate limiting):
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/sentry-raven-2.7.4/lib/raven/transports/http.rb:34:in `rescue in send_event'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/sentry-raven-2.7.4/lib/raven/transports/http.rb:16:in `send_event'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/sentry-raven-2.7.4/lib/raven/client.rb:37:in `send_event'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/sentry-raven-2.7.4/lib/raven/instance.rb:81:in `send_event'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/sentry-raven-2.7.4/lib/raven/instance.rb:126:in `capture_type'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/sentry-raven-2.7.4/lib/raven/integrations/rake.rb:9:in `display_error_message'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:195:in `rescue in standard_exception_handling'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:185:in `standard_exception_handling'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:80:in `run'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
D:/Ruby25-x64/bin/rake:31:in `load'
E, [2018-10-28T12:18:49.479361 #16532] ERROR -- sentry: ** [Raven] Failed to submit event: NameError: uninitialized constant DoorkeeperSequel::ConfidentialApplicationsGenerator
rake aborted!
NameError: uninitialized constant DoorkeeperSequel::ConfidentialApplicationsGenerator
D:/Ruby25-x64/lib/ruby/gems/2.5.0/bundler/gems/doorkeeper-sequel-945dc72250c8/lib/doorkeeper-sequel/tasks/doorkeeper-sequel.rake:25:in `block (3 levels) in <top (required)>'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/task.rb:271:in `block in execute'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/task.rb:271:in `each'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/task.rb:271:in `execute'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/task.rb:213:in `block in invoke_with_call_chain'
D:/Ruby25-x64/lib/ruby/2.5.0/monitor.rb:226:in `mon_synchronize'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/task.rb:193:in `invoke_with_call_chain'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/task.rb:182:in `invoke'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:160:in `invoke_task'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:116:in `block (2 levels) in top_level'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:116:in `each'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:116:in `block in top_level'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:125:in `run_with_threads'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:110:in `top_level'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:83:in `block in run'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:186:in `standard_exception_handling'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/lib/rake/application.rb:80:in `run'
D:/Ruby25-x64/lib/ruby/gems/2.5.0/gems/rake-12.3.1/exe/rake:27:in `<top (required)>'
D:/Ruby25-x64/bin/rake:31:in `load'
D:/Ruby25-x64/bin/rake:31:in `<main>'
Tasks: TOP => doorkeeper_sequel:generate:confidential_applications
```

The weird thing is, I can see these tasks in `rake --tasks`.

What could be the reason?

Note: Against `v2.0` and  `master` tested.

### PR DESCRIPTION
**Pull request by nbulaj**
Fixes `uninitialized constant DoorkeeperSequel::ConfidentialApplicationsGenerator`

### CODE REVIEW

File: lib/doorkeeper-sequel/mixins/application_mixin.rb [hunk 1]
```diff
@@ -70,29 +72,11 @@ def by_uid(uid)
...
-      end
-
       def column_names
         columns.map(&:to_s)
       end
     end
 
-    # Fallback to existing, default behaviour of assuming all apps to be
-    # confidential if the migration hasn't been run
-    def confidential
```
Comment (gencer) [inline lib/doorkeeper-sequel/mixins/application_mixin.rb:84]: @nbulaj, Is there a specific reason for removing this and above methods? I updated my gemfile and i got

```NameError: undefined method `confidential' for class `Doorkeeper::Application'```

at startup.

Comment (nbulaj) [reply lib/doorkeeper-sequel/mixins/application_mixin.rb:84]: You didn't create a db field ? Because for Doorkeepeer 5.0 it exists always

Comment (gencer) [reply lib/doorkeeper-sequel/mixins/application_mixin.rb]: Oh man, No. This might be the reason for #16. Let me test.

Comment (gencer) [reply lib/doorkeeper-sequel/mixins/application_mixin.rb:84]: Yes I did created. This error is from api only project. (Wrong alarm). I still have #16.

**Pull request merged**